src/models/editorArguments.interface.ts

```diff
@@ -2,8 +2,9 @@
 import { Column, ElementPosition, PositionMethod } from './index';
 import type { SlickDataView } from '../slick.dataview';
 import type { SlickGrid } from '../slick.grid';
+import type { GridOption } from '../models/gridOption.interface';     
```

Comment:
similar as above, you are already inside `models/` folder, no need to get out and get back in. However, you should remove this line and instead modify the import on line 2, so please modify line 2 as suggested below

```suggestion
import type { Column, ElementPosition, GridOption, PositionMethod } from './index';
```
also notice that I forgot to add the `type` so please make sure you add it. It is better for import and build perf